### PR TITLE
Right script fixes

### PR DESCRIPTION
**Pull request by douglaswth**
- Allow `GET /api/right_scripts/:id/source` to work since it comes back as `text/plain`.

### CODE REVIEW

File: lib/right_api_client/client.rb [hunk 1]
```diff
@@ -369,7 +369,11 @@ def do_get(path, params={})
       data = if resource_type == 'text'
         { 'text' => body }
       else
-        JSON.parse(body, :allow_nan => true)
+        if res && res.headers[:content_type].split(';').first.strip == 'text/plain'
+          {:value => body}
```
Comment (EfrainOlivares) [inline lib/right_api_client/client.rb]: just a question on style:  spaces between the text and brackets?  Other hashes have them in this file.

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .DS_Store
 nbproject
-*login.yml
+*login*.yml
```
Comment (EfrainOlivares) [inline .gitignore]: just curious.  why did you need the more restrictive ignore on the login file?

Comment (douglaswth) [inline .gitignore]: More restrictive? This is less restrictive, now it ignores more login files!

**Comment by rtlong** [comment]
Woot! This is great. Would love to see this out in a released version!
